documentation/src/main/asciidoc/topics/proc_creating_server_security_realms.adoc

```diff
@@ -50,7 +50,13 @@ You must manually synchronize credentials in a properties realm to each node in
 . Use the `security-realms` element in the `security` configuration to contain create multiple security realms.
 . Add a security realm with the `security-realm` element and give it a unique name with the `name` attribute.
 +
-To follow the example, create one security realm named `application-realm` and another named `management-realm`.
+[Important]
```

Comment:
[IMPORTANT]

Reply:
Of course, I do not specify a snippet for this type of admonition.